src/packages/recompose/branch.js

```diff
@@ -2,7 +2,11 @@ import React from 'react'
 import createHelper from './createHelper'
 import createEagerFactory from './createEagerFactory'
 
-const branch = (test, left, right) => BaseComponent =>
+const branch = (
+  test,
+  left,
+  right = baseComponent => baseComponent
```

Comment:
This is a bit hard to read. Could you define `identity` above and use that instead?